src/c/sk_bitmap.cpp

```diff
@@ -93,6 +101,15 @@ bool sk_bitmap_ready_to_draw(sk_bitmap_t* cbitmap) {
     return AsBitmap(cbitmap)->readyToDraw();
 }
 
+bool sk_bitmap_compute_is_opaque(sk_bitmap_t* cbitmap) {
+    return SkBitmap::ComputeIsOpaque(*AsBitmap(cbitmap));
+}
+
+const sk_pixmap_t* sk_bitmap_get_pixmap(sk_bitmap_t* cbitmap) {
+    const SkPixmap& pixmap = AsBitmap(cbitmap)->pixmap();
+    return ToPixmap(&pixmap);
+}
+
```

Comment:
This is already exposed as `PeekPixels()`. Does this add any extra features or can we rather use that method?

Reply:
this method ALWAYS returns the internal pixmap even if it contains no pixels (addr == nullptr)

Reply:
```cpp
bool SkBitmap::peekPixels(SkPixmap* pmap) const {
    if (this->getPixels()) {
        if (pmap) {
            *pmap = fPixmap;
        }
        return true;
    }
    return false;
}
```

```cpp
// BEGIN
    SkPixmap src;
    
    // if this returns true, it is assigned fPixmap, which is the return value of pixmap()
    // this returns true only if getPixels() != nullptr

    // getPixels() returns fPixmap.writable_addr()
    // both addr and writable_addr return fPixels

    // HOWEVER, on failure, this avoids assigning the pixmap

    if (!this->peekPixels(&src)) {
```

Reply:
Makes sense.